### ISSUE TITLE
add documentation and/or possibility for failover/clustering

### ISSUE THREAD

**Issue by gnur** [influxdata/kapacitor#277]
I cannot find any documentation which shows how to make kapacitor clustered or some kind of master slave setup.

If it is possible to make kapacitor clustered, the documentation is severely lacking, if it isn't possible, this is a major feature that I would love to see added to kapacitor.

**Comment by nathanielc** [comment]
This is a good idea, basically to cluster Kapacitor in its current state there are too options:

* Active-Active: Run two Kapacitor instances doing the same thing and dedup alerts etc on the backend.
* Active-Passive: Place an consistent load-balancer in front of Kapacitor and failover all traffic to the passive instance if the active instance has issues.

We should put together a good write up on how one could do either of these and which is better for differing use cases.

**Comment by yosiat** [comment]
@nathanielc I am wondering, how can you implement the "Active-Passive" approach if you have stateful alerts like window? If you switch to the passive then you will stream of alerts as "OK" and only after sometime as "Critical" (by the way, we have alerts with window of large periods - hours).

So we chose the "Active-Active" approach and choose a "leader" to take from the alerts, the "leader choosing" will be by the instance with the largest uptime.

What do you think?

**Comment by beckettsean** [comment]
@nathanielc how do you feel about closing this in favor of the docs repo issue I just opened?

**Comment by nathanielc** [comment]
@beckettsean Works for me. Thanks
